airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -151,6 +156,7 @@ def get_dag_runs_batch(session):
                                                           total_entries=total_entries))
 
 
+@security.requires_authentication
```

Comment:
Yes, We can configure it on a higher layer, but prefer to be explicit than implicit, because we have two views that do not require authorization - `/health` and `/version`.

Reply:
Perfect!

Reply:
if no auth is the exception here (i.e. only /health and /version), wouldn't it be safer to make auth required the default, then turn off auth specifically for those two endpoints?

Reply:
I was based on how the experimental API is realized, but I will prepare a patch that will update the stable and experimental API.